app/controllers/api/messages_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::MessagesController < ApplicationController
+  def index
+    ids = Message.ids
+    random_greeting_id = ids.sample
+    random_greeting = Message.find(random_greeting_id).greeting
+    render json: { message: random_greeting }
+  end
```

Comment:
- [ ] Nice job but your app is not fetching any message you need to add `**greeting messages**` to your table.

Reply:
@dasileker 
I've greeting messages in my table , but I think you need to add some messages in your installation when you are running my application in order to see some random messages.

Reply:
@Taher-web-dev the roll to create an api is to add the messages and fetch the messages in react.

Reply:
@dasileker ,
The messages are created in my local machine, and I'm fetching the API locally. I can't understand what do you mean?

Reply:
- [ ] okay but the requirement of this is to add them to your data, the API should not be empty and here it comes the meaning of fetch because it fetches the data inside not add the data to the API.

![Capture web_22-3-2022_105613_github com](https://user-images.githubusercontent.com/61562626/159454738-4153a59a-6051-4e74-af40-3b3d2856d7d5.jpeg)

Reply:
@dasileker 
Yeah, all the requirements were implemented. I will add some messages using seed file, but you need to add them also to your local machine using db:seed .